android/src/main/java/com/github/yamill/orientation/listeners/OrientationConfigListener.kt

```diff
@@ -12,28 +12,23 @@ import com.facebook.react.bridge.LifecycleEventListener
 import com.facebook.react.bridge.ReactContext
 import com.facebook.react.common.ReactConstants
 import com.facebook.react.modules.core.DeviceEventManagerModule
+import com.github.yamill.orientation.util.OrientationUtil
 
 class OrientationConfigListener internal constructor(
-    reactContext: ReactContext,
+    private val reactContext: ReactContext,
     private val onGetCurrentActivity: () -> Activity?,
 ) : LifecycleEventListener {
 
     private val receiver = object : BroadcastReceiver() {
 
         override fun onReceive(context: Context, intent: Intent) {
             val newConfig = intent.getParcelableExtra<Configuration>(INTENT_VALUE_KEY)!!
-            val orientationValue = if (newConfig.orientation == 1) {
+            val orientation = if (newConfig.orientation == 1) {
```

Comment:
nit: This works, but another option I was thinking of was to pass the orientation Int to `tryEmitOrientationChange`. Then `tryEmitOrientationChange` can check if it's UNKNOWN or null. If it's neither, then it can emit the change.

Reply:
Right now, it looks like `onReceive` cannot emit `UNKNOWN`, but `onHostResume` can emit `UNKNOWN`. Doing the check in `tryEmitOrientationChange` will keep these two scenarios more consistent with each other.